<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabWidget</class>
 <widget class="QTabWidget" name="TabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>28</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
  </property>
  <property name="currentIndex">
   <number>1</number>
  </property>
  <widget class="QWidget" name="tab">
   <attribute name="title">
    <string>拨号按键</string>
   </attribute>
   <widget class="QPushButton" name="pushButton_star">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>550</y>
      <width>180</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(141, 141, 141, 255), stop:0.534091 rgba(0, 0, 0, 255), stop:1 rgba(148, 148, 148, 255));
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_number9">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>430</y>
      <width>180</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(141, 141, 141, 255), stop:0.534091 rgba(0, 0, 0, 255), stop:1 rgba(148, 148, 148, 255));
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_number1">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>190</y>
      <width>180</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(141, 141, 141, 255), stop:0.534091 rgba(0, 0, 0, 255), stop:1 rgba(148, 148, 148, 255));
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_phoneNumber">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>540</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>58</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(184, 184, 184, 255));</string>
    </property>
    <property name="text">
     <string>123 4567 8901</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_number4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>310</y>
      <width>180</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(141, 141, 141, 255), stop:0.534091 rgba(0, 0, 0, 255), stop:1 rgba(148, 148, 148, 255));
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_number6">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>310</y>
      <width>180</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(141, 141, 141, 255), stop:0.534091 rgba(0, 0, 0, 255), stop:1 rgba(148, 148, 148, 255));
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sign">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>550</y>
      <width>180</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(141, 141, 141, 255), stop:0.534091 rgba(0, 0, 0, 255), stop:1 rgba(148, 148, 148, 255));
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>#</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_number2">
    <property name="geometry">
     <rect>
      <x>575</x>
      <y>190</y>
      <width>180</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(141, 141, 141, 255), stop:0.534091 rgba(0, 0, 0, 255), stop:1 rgba(148, 148, 148, 255));
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_number5">
    <property name="geometry">
     <rect>
      <x>575</x>
      <y>310</y>
      <width>180</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(141, 141, 141, 255), stop:0.534091 rgba(0, 0, 0, 255), stop:1 rgba(148, 148, 148, 255));
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_number0">
    <property name="geometry">
     <rect>
      <x>575</x>
      <y>550</y>
      <width>180</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(141, 141, 141, 255), stop:0.534091 rgba(0, 0, 0, 255), stop:1 rgba(148, 148, 148, 255));
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_number3">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>190</y>
      <width>180</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(141, 141, 141, 255), stop:0.534091 rgba(0, 0, 0, 255), stop:1 rgba(148, 148, 148, 255));
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_callphone">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>285</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0.170455 rgba(0, 0, 0, 255), stop:1 rgba(121, 121, 121, 255));
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>拨号</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_number8">
    <property name="geometry">
     <rect>
      <x>575</x>
      <y>430</y>
      <width>180</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(141, 141, 141, 255), stop:0.534091 rgba(0, 0, 0, 255), stop:1 rgba(148, 148, 148, 255));
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_delete">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>60</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0.170455 rgba(0, 0, 0, 255), stop:1 rgba(121, 121, 121, 255));
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_number7">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>430</y>
      <width>180</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(141, 141, 141, 255), stop:0.534091 rgba(0, 0, 0, 255), stop:1 rgba(148, 148, 148, 255));
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="tab">
   <attribute name="title">
    <string>联系人</string>
   </attribute>
   <widget class="QPushButton" name="pushButton_Contacts_callPhone">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>285</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0.170455 rgba(0, 0, 0, 255), stop:1 rgba(121, 121, 121, 255));
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>拨号</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_Contacts">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>630</width>
      <height>580</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="tab_2">
   <attribute name="title">
    <string>通话记录</string>
   </attribute>
   <widget class="QPushButton" name="pushButton_recordCall">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>285</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0.170455 rgba(0, 0, 0, 255), stop:1 rgba(121, 121, 121, 255));
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>拨号</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_records">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>630</width>
      <height>580</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="tab_4">
   <attribute name="title">
    <string>蓝牙音乐</string>
   </attribute>
   <widget class="QPushButton" name="pushButton_PreSong">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>440</y>
      <width>206</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(141, 141, 141, 255), stop:0.534091 rgba(0, 0, 0, 255), stop:1 rgba(148, 148, 148, 255));</string>
    </property>
    <property name="text">
     <string>上一首</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_play_or_stop">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>440</y>
      <width>206</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(141, 141, 141, 255), stop:0.534091 rgba(0, 0, 0, 255), stop:1 rgba(148, 148, 148, 255));</string>
    </property>
    <property name="text">
     <string>播放/暂停</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stop">
    <property name="geometry">
     <rect>
      <x>662</x>
      <y>440</y>
      <width>206</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(141, 141, 141, 255), stop:0.534091 rgba(0, 0, 0, 255), stop:1 rgba(148, 148, 148, 255));
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_nextSong">
    <property name="geometry">
     <rect>
      <x>868</x>
      <y>440</y>
      <width>206</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(141, 141, 141, 255), stop:0.534091 rgba(0, 0, 0, 255), stop:1 rgba(148, 148, 148, 255));
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>下一首</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_vol">
    <property name="geometry">
     <rect>
      <x>1074</x>
      <y>440</y>
      <width>206</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(141, 141, 141, 255), stop:0.534091 rgba(0, 0, 0, 255), stop:1 rgba(148, 148, 148, 255));
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>音量</string>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_vol">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>280</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">selection-background-color: rgb(199, 199, 199);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="tab_3">
   <attribute name="title">
    <string>蓝牙设置</string>
   </attribute>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>540</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>蓝牙</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_open_or_close">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>60</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>开/关</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>120</y>
      <width>300</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 75 28pt &quot;ADMUI3Lg&quot;;
color: rgb(145, 145, 145);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;RD-1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>260</y>
      <width>600</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(230, 230, 230);</string>
    </property>
    <property name="text">
     <string>可用设备</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_disconnect">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>430</y>
      <width>60</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>断开</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>180</y>
      <width>600</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(213, 213, 213);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>300</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>设备名称</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>410</y>
      <width>600</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(230, 230, 230);</string>
    </property>
    <property name="text">
     <string>已配对设备</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>600</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(213, 213, 213);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_MF_device">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>440</y>
      <width>521</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_SF_Dev">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>290</y>
      <width>591</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
